javascript/app-model-exposure/src/hooks/bindData/useDefinitionDropdownOptions.ts

```diff
@@ -17,6 +16,15 @@ import { CommandActionType } from '../../types/workflow/CommandActionType.js'
 import { getDefinitionsByType } from '../../utils/definition.js'
 import { useCausalEffects } from '../causalEffects.js'
 
+export const typeLabel = {
+	[CausalFactorType.CauseExposure]: 'Causes Exposure',
```

Comment:
We usually label with the style "Capital not capital" per fluent guidelines, so these would be "Causes exposure" and "Causes outcome"